tests/performance/performance_suite_test.go

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 VMware
```

Comment:
Do we want this to live under tests? This isn't something we are planning on running regularly. Maybe it can be its on go module somewhere?

Reply:
Sure. Any objection to moving it to top level package `benchmark`?

Reply:
When you say a top level package, do you mean under `/pkg` or at the same level as `/pkg`? If the latter, I'd be ok with that if it's its own go module.